src/components/Selected/Selected.styles.ts

```diff
@@ -1,12 +1,13 @@
 import styled from "styled-components";
+import { LanguageProps } from "./Selected";
 
 export const Container = styled.div``;
 export const Percentage = styled.span`
   color: ${({ theme }) => theme.colors.primary};
 `;
-export const Language = styled.a`
-  cursor: pointer;
-  text-decoration: underline;
+export const Language = styled.a<LanguageProps>`
+  cursor: ${({ disabled }) => (disabled ? undefined : "pointer")};
+  text-decoration: ${({ disabled }) => (disabled ? undefined : "underline")};
```

Comment:
i'd rather use a short circuit in both conditions instead of passing `undefined`